code/modules/clothing/gloves/color.dm

```diff
@@ -12,6 +12,52 @@
 	custom_price = 1200
 	custom_premium_price = 1200
 
+/obj/item/toy/sprayoncan
+	name = "spray-on insuls applicator"

```

Comment:
Might want to go with a formal name than this one, like just "spray-on insulation applicator".

Reply:
Seeing as this is venture capitalism the machine, something a bit more snappy and marketable would probably be better. Think those god awful Billy Mays ads.